modules/evaluating-rag-system-quality-with-ragas.adoc

```diff
@@ -0,0 +1,66 @@
+:_module-type: PROCEDURE
+
+[id="evaluating-rag-system-quality-with-Ragas_{context}"]
+= Evaluating RAG system quality with Ragas metrics
+
+[role='_abstract']
+Evaluate your RAG system quality by testing your setup, using the example provided in the link:https://github.com/trustyai-explainability/llama-stack-provider-ragas/blob/main/demos/basic_demo.ipynb[demo notebook], which outlines the basic steps for evaluating your RAG system with Ragas using the Python client. You can execute the demo notebook steps from a Jupyter environment. 
+
+Alternatively, you can submit an evaluation by directly using the `http` methods of the link:{odhdocshome}/working_with_llama_stack/overview-of-llama-stack_rag#the_llamastackdistribution_custom_resource_api_providers[Llama Stack API]. 
+// (link to the llama stack 0.3.0 docs -- unknown at this time Sara to check with Kelly and Chris on this. Potentially this link? https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.25/html/working_with_llama_stack/overview-of-llama-stack_rag#the_llamastackdistribution_custom_resource_api_providers).
+
+[IMPORTANT]
+====
+The Llama Stack pod must be accessible from the Jupyter environment in the cluster, which may not be the case by default.
```

Comment:
? what do they do if it's not accessible - is there doc that describes how to make it accessible? add a link here